hack/test-stirunimage.sh

```diff
@@ -34,16 +34,13 @@ fi
 
 trap cleanup EXIT SIGINT
 
-echo
-echo 
-echo
-
-sti build git://github.com/bparees/openshift-jee-sample openshift/wildfly-8-centos test-jee-app --run=true &> "${STI_ROOT}/hack/sti-run.log" &
+echo "Running 'sti build --run=true ...'"
+s2i build git://github.com/bparees/openshift-jee-sample openshift/wildfly-8-centos test-jee-app --run=true &> "${STI_ROOT}/hack/sti-run.log" &
```

Comment:
This should be also fixed to use the `openshift/wildfly-81-centos7` instead, can be part of that refactor of yours.